ci/nodepool/scripts/bootstrap.sh

```diff
@@ -6,7 +6,14 @@ export DISTRIBUTION=$(python -c "import platform, sys
 sys.stdout.write(platform.dist()[0])")
 export DISTRIBUTION_MAJOR_VERSION=$(python -c "import platform, sys
 sys.stdout.write(platform.dist()[1].split('.')[0])")
-export PKG_MGR=$(which dnf 2>/dev/null || which yum 2>/dev/null)
+
+# use dnf if you can, otherwise use yum
+if dnf --version; then
```

Comment:
I'm happy to redirect it, but the silence of the line I'm replacing made it a little difficult to debug. Being able to see that dnf either printed a version or was not found makes it quite clear in the image building log why dnf was (or was not) chosen.